service-type-database/build-db

```diff
@@ -1,4 +1,4 @@
-#!@PYTHON@
+#!/usr/bin/env python
```

Comment:
python3 ?!

Reply:
The use of /usr/bin/env is an unrelated change and should be dropped from this pull request or at least be moved into a separate commit with a proper justification for this change. Distro packages are stronly discouraged to use /usr/bin/env, as this can cause them to pick up a "random" python version. So, instead of changing this to "/usr/bin/env python3", please drop this hunk completely.

Reply:
https://www.debian.org/doc/packaging-manuals/python-policy/ch-python.html#s-interpreter

Reply:
For what it is worth, Fedora has similar guidelines: https://fedoraproject.org/wiki/Packaging:Python#Multiple_Python_Runtimes

Reply:
Distributions can edit the `#!` line according to their preferred policies at install time if they need to (in Debian, `dh-python` does that), so this is not necessarily a guideline that upstreams need to follow.

(But yes, it would have been better if this change was in a separate commit where it could be justified and merged (or not) on its own merits.)